tests/suites/test_suite_asn1parse.function

```diff
@@ -614,7 +614,7 @@ void get_sequence_of( const data_t *input, int tag,
             cur = &head;
             while( *rest )
             {
-                ++mbedtls_test_info.step;
+                mbedtls_test_set_step( mbedtls_test_info.step + 1 );
```

Comment:
(Not a blocker, I intend this as a discussion for a possible follow-up, I'm already relying on this PR in my ongoing work so I don't want to delay it.) I'm not convinced that using the write abstraction and reading from the global variable directly in the same instruction is the best way to express this. At least with `++mbedtls_test_info.step` you could instantly see that this was incrementing a number, whereas now it looks like some number is being set to one plus some other number. Should there be a `mbedtls_test_get_step()`? Or a `mbedtls_test_next_step( )` (incrementing the step at each go through a loop is common)? Or should we change the code here to `mbedtls_test_set_step( rest - description )` (which would change the step to count positions in the string instead of loop iterations)? Or should we add a local variable to count loop iterations?

Reply:
I vote for mbedtls_test_set_step( step ) where step counts loop iterations. And please do it now, we should be able to merge this today anyway.

Reply:
Yes I agree. I think this makes the most logical sense and I will update now as it should not take long to do.

Reply:
Done.